.drone.yml

```diff
@@ -107,6 +107,7 @@ steps:
   image: casperlabs/node-build-u1804
   commands:
   - make setup
+  - make build-contracts
```

Comment:
This should not be required.  Rather, the new unit test should be decorated with `#[ignore]`, and it will thus not be executed as part of the `make test` call, but will be executed as part of the `make test-contracts` call, where `make test-contracts` already invokes `make build-contracts`.

Reply:
Aha! Thanks for pointing that out.